src/app/shared/header.js

```diff
@@ -12,7 +12,7 @@ class Header extends React.Component {
 			<div className='nav-options'>
 				<img src='http://www.colorado.edu/cwa/profiles/express/themes/culayers/logo.png' />
 				<ul>
-					<li><a>Home</a></li>
+					<li><a><i className="fa fa-home fa-3x"></i></a></li>
```

Comment:
Just going from what we talked about in inclusive design, can you reduce this to a fa fa-home fa-2x